lib/src/main/java/com/auth0/jwt/algorithms/ECDSAAlgorithm.java

```diff
@@ -49,7 +47,7 @@ public void verify(DecodedJWT jwt) throws SignatureVerificationException {
             if (!valid) {
                 throw new SignatureVerificationException(this);
             }
-        } catch (NoSuchAlgorithmException | SignatureException | InvalidKeyException | IllegalStateException e) {
+        } catch (Exception e) {
```

Comment:
expand the catch clause instead of capturing a generic `Exception`. Capture each of them and make all do the same. (Throw a wrapped exception in this particular case)